components/atoms/Select/single-select.tsx

```diff
@@ -60,7 +60,7 @@ const SingleSelect = ({
       <DropdownMenuTrigger
         data-inset-label={insetLabel}
         className={clsx(
-          "flex text-sm px-3 py-1.5 !border !border-slate-200 rounded-md bg-white data-[state=open]:border-orange-500 min-w-max",
+          "flex w-full md:w-fit text-sm px-3 py-1.5 !border !border-slate-200 rounded-md bg-white data-[state=open]:border-orange-500 min-w-max",
```

Comment:
I took this for a spin on larger screens and I see the fix, but on smaller screens, there's still a gap between the arrow and the dropdown component. Adding a `justigy-between` should fix it, but take it for a spin on the different screen sizes.

![CleanShot 2024-09-03 at 14 30 44](https://github.com/user-attachments/assets/1b3fb3b3-936a-4d0d-9656-6538ece98d1d)

```suggestion
          "flex w-full justify-between md:w-fit text-sm px-3 py-1.5 !border !border-slate-200 rounded-md bg-white data-[state=open]:border-orange-500 min-w-max",
```

![CleanShot 2024-09-03 at 14 37 10](https://github.com/user-attachments/assets/ae06ef10-20f5-4215-837b-03473f149827)